ansible/roles/stack-sunbird/templates/sunbird_lms-service.env

```diff
@@ -117,3 +117,4 @@ managed_user_limit={{managed_user_limit | default(30)}}
 
 adminutil_base_url = {{adminutil_base_url}}
 adminutil_sign_endpoint = {{adminutil_sign_endpoint | default('v1/sign/payload')}}
+sunbird_user_qrcode_courses_limit=5000
```

Comment:
we should have a variable to override this and add 5000 as default @maheshkumargangula

Reply:
@rjshrjndrn  resolved